_data/sites.json

```diff
@@ -8612,5 +8612,15 @@
         "domains": [
             "zynga.com"
         ]
+    },
+    
```

Comment:
```suggestion

```

Reply:
Is there a way to revert this commit or would it be better for me to just make a new one?